static/src/javascripts/projects/common/modules/navigation/membership.js

```diff
@@ -55,7 +55,9 @@ const showAccountDataUpdateWarningMessage = accountDataUpdateWarningLink => {
 const updateLink = accountDataUpdateWarning();
 
 const canShow: () => Promise<boolean> = () =>
-    Promise.resolve(updateLink !== null);
+    Promise.resolve(
+        updateLink !== null && hasUserAcknowledgedBanner(messageCode)
```

Comment:
Should this be `!hasUserAcknowledgedBanner(messageCode)`? As in `canShow` should resolve with `true` if the user has *not* already acknowledged the banner?